css/enhancedsteam.css

```diff
@@ -1870,3 +1870,13 @@ input[type=checkbox].es_dlc_selection:checked + label {
 	margin-top: 0 !important;
 }
 
+/***************************************
+ * Apps community hub
+ * add_app_page_wishlist()
+ **************************************/
+.apphub_StoreInfoHeader.es_highlighted .price,
```

Comment:
Please note the the class name is "es_highlight**_ed_**" to accommodate for #1106